camera_alignment_core/alignment_utils/segment_rings.py

```diff
@@ -133,7 +133,12 @@ def segment_cross(
                 ):
                     break
 
+        # try:
```

Comment:
intentionally keeping this commented out?  We usually go with actually deleting things rather than commenting out.  I suspect this is a testing thing you meant to remove.